README.md

```diff
@@ -20,7 +20,7 @@ Install from GitHub:
 ```bash
 git clone https://github.com/GeneDx/phenopy.git
 cd phenopy
-python setup.py install
+poetry install
```

Comment:
It's a nice practice, though not strictly necessary, to install poetry in the global environment. My preferred way is to use pipx. pipx can also install different versions with a suffix so you could have `poetry-1.2.5` for testing upgrade changes.

The issue with installing poetry with pip is that it can sometimes have conflicts with the package's deps - though I've only rarely seen this.